### DISCUSSION TITLE
Compare to FontAwesome

### DISCUSSION THREAD

**Discussion by jamesli2021** [tabler/tabler-icons#370] [General]
Which has over 19,000 icons, and Figma supported FontAwesome. What is the strong points of choosing Tabler icons?

**Comment by codecalm** [comment]
hi @jamesli2021! Tabler Icons are made by stroke width (like variable font), not by outline, so you can change it in any time. If one were to count like FontAwesome and the 6 strokes (1px, 1.25px, 1.5px, 1.75px, 2px and 2.25px) Tabler Icons has a total of 18,000 icons :)

And are, of course, open source unlike Font Awesome :)

**Reply by jamesli2021** [reply]
I would called it Awesome Tabler! Appreciate.
